### DISCUSSION TITLE
spegnere il termostato quando tutte le valvole si chiudono

### DISCUSSION THREAD

**Discussion by maurominella** [indomus/forum#1816] [HUB: Home Assistant]
Ho 5 termosifoni ciascuno con la propria valvola termostatica configurata in Home Assistant, e tutte funzionano correttamente.
In più ho un termostato centrale della casa, sempre configurato in Home Assistant, che posso accendere / spegnere o modificare la temperatura limite. Poiché la temperatura è regolata su ogni termosifone, ho impostato il termostato per evitare che si spenga, quindi con una temperatura molto alta (tipo 28 gradi), altrimenti rischierebbe di spegnere la caldaia centrale anche se una stanza è ancora fredda, solo perché il termostato si trova in una zona più calda.
Quando una valvole registrano una temperatura superiore a quella impostata, naturalmente si spenge. 

Il problema però è che quando TUTTE le valvole si spengono, la caldaia va in blocco perché è accesa ma nessun termosifone chiede acqua.

Come posso fare in modo che quando TUTTE le valvole sono spente, si spenga anche il termostato centrale (e quindi la caldaia)? Il segnale di spegnimento al termostato lo so inviare (anche usando Home Assistant), ma il problema è che questo segnale deve essere inviato non appena TUTTE le valvole si chiudono, e naturalmente va riacceso quando anche SOLO UNA VALVOLA si riaccende.

Grazie

**Comment by 50494554524F** [comment]
metti le gli switch/binary_sensor delle valvole in un gruppo di tipo switch/binary_sensor 

https://www.home-assistant.io/integrations/group/#yaml-configuration

**Comment by maurominella** [comment]
Grazie per la risposta. Però forse non ho inteso bene, mi pare che questo metodo si usi per accendere/spegnere le valvole tutte insieme.  Come faccio a spegnere il termostato quando tutte le valvole si chiudono, e riaccenderlo quando solo una si accende?

**Reply by 50494554524F** [reply]
il gruppo diventa on se una valvola diventa on, diventa off se tutte off... poi userai automazione per la caldaia

**Reply by Scialla** [reply]
Occhio che l'impianto ha bisogno di sfogare, il ricircolo deve essere sempre attivo, a meno che la caldaia non abbia la funzione di ricircolo interno, altrimenti la caldaia va in protezione e si blocca.

Io ho tolto una valvola da un termosifone e ho messo il termostato in quella stanza e, ora, tutto funziona perfettamente

**Reply by maurominella** [reply]
Sì, appunto per questo io voglio spegnere la caldaia quando tutti i termosifoni chiudono le loro valvole.
Certo potrei evitare questo compito lasciando un calorifero sempre acceso al massimo, ma mi sembrerebbe davvero un fallimento: ho acquistato tutti i prodotti comandabili attraverso home assistant, e poi lascio una stanza con il calorifiero sempre acceso? Trovo almeno 4 inconvenienti a fare questa cosa: 
- chi si trova nella stanza senza valvola sul calorifero potrebbe scoppiare di caldo, 
- spesa in più,
- produzione di CO2 inutile,
- se la caldaia si spegne usando il termostato centrale, non possiamo essere certi che tutte le stanze abbiano raggiunto la temperatura desiderata.

Il suggerimento di creare un gruppo che automaticamente diventa ON se una sola valvole è accesa, e automaticamente va in OFF se è spenta, abbinato all'utilizzo di una procedura di automazione (che però non ho ancora sperimentato), mi sembra davvero l'uovo di colombo per consumare il minimo possibile e ottenere la temperatura voluta in tutte le stanze.

**Reply by Scialla** [reply]
Ci avevo pensato anche io ma, il giorno che HA va down ti si fotte la caldaia...

Per me il riscaldamento deve funzionare autonomamente e HA deve essere un valore aggiunto, e non indispensabile al funzionamento del riscaldamento stesso...
Io ho tolto la valvola nella stanza di mia figlia e, considerato che normalmente tengo sui 19 gradi, lei è felicissima di questa cosa :)

Con HA ho fatto automazioni che, ad esempio, mi chiudono la valvola della stanza in cui viene aperta la porta/finestra, automazioni che, anche dovessero venire a mancare, non comprometterebbero il funzionamento del riscaldamento.

Quando mi ero accorto del problema tipo il tuo, la caldaia ha fatto dei rumoracci enormi prima di andare in blocco; quanti ne potrebbe reggere prima di spaccarsi?
A sto punto cambia caldaia e prendine una col ricircolo interno....

Per il tuo 4 punto ho impostato che il termostato rimane aperto se una valvola chiama, quindi tutte le stanze raggiungono la temperatura loro impostata

Io ho tutto Netatmo e ho preferito gestire la stanza di mia figlia col termostato rimuovendo la valvola, e lei ne è ben contenta ;)

**Reply by 50494554524F** [reply]
tenere un termosifone sempre aperto non guasta; l'ingresso, con il portone e gli spifferi è il miglior candidato... più che altro per evitare uno sbalzo di pressione per la pompa della caldaia prima che si autoregoli. inoltre, per evitare il rumore, andrebbe ridotta la velocità dell'acqua...
